server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/json/Serializer.scala

```diff
@@ -34,7 +34,7 @@ import org.apache.james.mailbox.model.{MailboxACL, MailboxId}
 import play.api.libs.functional.syntax._
 import play.api.libs.json._
 
-import scala.collection.{Seq => JsonSeq}
+import scala.collection.{Seq => MutableSeq}
```

Comment:
i don't think it's mutable, as it is not from `scala.collection.mutable`

Reply:
Ah yeah true... `CollectionSeq` maybe ?

Reply:
maybe :P, i don't know why they choosed to use this base type. As in scala 2.13. The default collection type in scala.predef is the immutable one. Maybe just some legacy.

Reply:
(maybe just using the FQCN could do the trick to not be ambiguous but not need to name it?)

Reply:
Probably some legacy yes I don't know... I could not find an other way though

Reply:
`LegacySeq`? :P

Reply:
I'd propose to use `scala.collection.Seq` to makes things explicit